fdb-record-layer-core/src/test/java/com/apple/foundationdb/record/provider/foundationdb/FDBExceptionsTest.java

```diff
@@ -59,7 +66,13 @@
  * Tests for {@link FDBExceptions}.
  */
 @Tag(Tags.RequiresFDB)
+@Execution(ExecutionMode.CONCURRENT)
 class FDBExceptionsTest {
+    @Nonnull
+    public static final String COMPLETION_EXCEPTION_HELPER_LOCK = "CompletionExceptionLogHelper";
```

Comment:
There are several tests in this class that modify a static variable. They use the resource lock feature (with this lock) to ensure that those tests in particular do not run concurrently.